imports/client/components/ProfilePage.tsx

```diff
@@ -51,9 +54,9 @@ const ProfilePage = (props: RouteComponentProps<ProfilePageParams>) => {
         discordAccount: undefined,
         muteApplause: undefined,
       },
-      viewerCanMakeOperator: Roles.userHasPermission(Meteor.userId(), 'users.makeOperator'),
-      viewerIsOperator: Roles.userHasRole(Meteor.userId()!, 'operator'),
-      targetIsOperator: Roles.userHasPermission(uid, 'users.makeOperator'),
+      viewerCanMakeOperator: deprecatedUserMayMakeOperator(Meteor.userId()),
```

Comment:
My best idea for how to make these not rely on the deprecated global calls is to add the route `/hunts/:huntId/hunters/:userId` for hunt-specific user lookup and then only present hunt-specific operator-y context on that page.

Then, the global user pages at `/users/:userId` could drop the operator-specific components.

Reply:
Ugh. We don't need to solve this now but that seems messy. It's already pretty messy that there are 2 different ways to get to the profile list that look the same but show different things

Reply:
Yeah -- I think we should probably nudge UI towards the hunt-scoped option.

I think if we're thinking about unrelated entities having accounts, it'd be pretty reasonable to restrict usage of `/users` and `/users/:userId` to admins and make everything else be hunt-scoped.  Might need to relocate `OwnProfilePage` to somewhere a little more generic.